nutkit/protocol/responses.py

```diff
@@ -495,6 +495,18 @@ def __init__(self, inUse, idle):
         self.idle = idle
 
 
+class EagerResult:
+    def __init__(self, keys, records, summary):
+        self.keys = keys
+        record_list = []
+        if records is not None:
+            for record in records:
+                record_list.append(Record(values=record["values"]))
+
+        self.records = record_list
```

Comment:
```suggestion
        self.records = [Record(**record) for record in records or []]
```

Difference: 
 * Will pass all elements of `records` to `Record`. But there shouldn't be any extra ones, should there?
 * Shorter and most likely more performant.